packages/spatial/src/physics/components/RigidBodyComponent.ts

```diff
@@ -76,7 +76,8 @@ export const RigidBodyComponent = defineComponent({
       linearVelocity: proxifyVector3(this.linearVelocity, entity),
       angularVelocity: proxifyVector3(this.angularVelocity, entity),
       /** If multiplier is 0, ridigbody moves immediately to target pose, linearly interpolating between substeps */
-      targetKinematicLerpMultiplier: 0
+      targetKinematicLerpMultiplier: 0,
+      canSleep: true /** Not a reactive property. Only on Init*/
```

Comment:
since this is an API, it should be moved above the `// internal` comment

Reply:
Oh Sorry I understood it incorrectly. Thought internal was everything bellow

Reply:
Ohhhh. Nevermind. Double misunderstood.